blueprints/batch-jobs/README.md

```diff
@@ -0,0 +1,214 @@
+# Karpenter Blueprint: Protecting batch jobs during the consolidation process
+
+## Purpose
+Karpenter can actively reduce the cluster cost by identifying when nodes can be removed or replaced because they are empty or there are a cheaper one available after some workload change. This process is called [consolidation](https://karpenter.sh/preview/concepts/disruption/#consolidation), and it implies the disruption of pods that are running in the node, if any, as they need to be rescheduled into another node. In some cases, like when running long batch jobs, you don't want those pods to be interrupted. You want to run them from start to finish without disruption, and replace or delete the node once they finished. To achieve that, you can set the `karpenter.sh/do-not-disrupt: "true"` annotation on the pod (more information [here](https://karpenter.sh/preview/concepts/disruption/#pod-level-controls)). By opting pods out of this disruption, you are telling Karpenter that it should not voluntarily remove a node containing this pod.
+
+## Requirements
+
+* A Kubernetes cluster with Karpenter installed. You can use the blueprint you have used to test this pattern at the `cluster` folder in the root of this repository.
+* A `default` Karpenter `NodePool` as that is the one you will use in this blueprint. You did this already in the ["Deploy a Karpenter Default EC2NodeClass and NodePool"](../../README.md) section from this repository.
+
+## Deploy
+You are going to use the `default` NodePool.
+
+If you want to first observe the default behaviour explained before, jump to [(Optional) Simulating the default behaviour](#(optional)-simulating-the-default-behaviour). 
+
+If you want to directly see how to avoid the interruption of jobs by the consolidation process, jump to [Preventing jobs of being evicted](#preventing-jobs-of-being-evicted).
+
+### (optional) Simulating the default behaviour
+This section simulates the default behaviour of the pods explained before, in which the Karpenter consolidation process interrupts the pods running the jobs, and re-schedule them into the cheaper node. To simulate it, deploy the [workloads-evicted yaml](/karpenter-blueprints/blueprints/batch-jobs/workloads-evicted.yaml):
+```
+$> kubectl apply -f workloads-evicted.yaml
+deployment.apps/nginx created
+job.batch/2-min-job created
+job.batch/5-min-job created
+```
+This will create three pods that require **11 vCPU** in total:
+-  NGINX server - 2 vCPU required
+-  2-minutes job - 7 vCPU required
+- 5-minutes job - 2 vCPU required
+
+Karpenter launches the cheapest EC2 instance for the workloads: a **c6g.4xlarge** on-demand instance (16 vCPU, 32 GiB). You can check this by executing:
```

Comment:
"Karpenter launches the cheapest EC2 instance for the workloads: a **c6g.4xlarge** on-demand instance" to "During this test, Karpenter decided to launch a **c6g.4xlarge** on-demand instance"